templates/monorepo/README.md

```diff
@@ -0,0 +1,82 @@
+# TypeScript Monorepo Template
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** I'd add some documentation about it to give developers a little context. Mainly any requirements/configurations, caveats, and little context about files in `.yarn/sdk` folder.
‏